CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+Contributing
+====================
+
+## Introduction
+
+First of all, thank you for taking the time to contribute!
+
+Before starting, make yourself familiar with the `nativescript-dev-webpack`'s [documentation](http://docs.nativescript.org/best-practices/bundling-with-webpack) and the official NativeScript [Contributing guide](https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md).
```

Comment:
Couple of suggestions:
 - Add a link to the NativeScript COC directly. The COC is valid for all repos in the NS organization so its OK to link it **directly** - this will spare you a click when you decide to read it.
 - Not sure the linking the tns-core-modules contributing guide is useful. It has some modules specific stuff. For example it contains **Submitting a PR** section which is covered here also. The **reporting bugs/issues** sections are more or less the same, but I think it is OK to just copy them here(maybe modify them a little with more links to the webpack docs). It will make the document easier to read.